mediapicker/src/main/java/com/mediapicker/gallery/presentation/fragments/PhotoCarousalFragment.kt

```diff
@@ -251,19 +274,23 @@ open class PhotoCarousalFragment : BaseFragment(), GalleryPagerCommunicator,
     }
 
     private fun setUpWithTabLayout() {
-        PagerAdapter(
-            childFragmentManager, listOf(
-                PhotoGridFragment.getInstance(
-                    getString(R.string.oss_title_tab_photo),
-                    getPhotosFromArguments()
-                ),
-                VideoGridFragment.getInstance(
-                    getString(R.string.oss_title_tab_video),
-                    getVideosFromArguments()
+        childView.findViewById<ViewPager>(R.id.viewPager).apply {
```

Comment:
childView.findViewById<ViewPager>(R.id.viewPager).apply { 
I thik it is not needed.

Reply:
done